bin/specextract

```diff
@@ -1135,18 +1070,21 @@ def event_stats(file, colname):
         src_x = max_cnts_src_pos.split(",")[0]
         src_y = max_cnts_src_pos.split(",")[1]
 
-        if colname=="x":
+        if colname == "x":
             return src_x
 
-        elif colname=="y":
+        elif colname == "y":
             return src_y
 
-        elif colname=="chipx":
+        elif colname == "chipx":
             return int(float(src_x))
 
-        elif colname=="chipy":
+        elif colname == "chipy":
             return int(float(src_y))
 
+    cr.__del__()
```

Comment:
We should not need this - there was some code way back in CIAO 4.<very-small-number> which did, but if you have evidence we still need this then we need a cratedev bug report!